waiter/docs/parameters.md

```diff
@@ -17,7 +17,7 @@ Additional (optional) parameters that can be set:
 |Parameter|Default Value|Valid Values|Description|Guidance|
 |---------|-------------|------------|-----------|--------|
 |`X-Waiter-Allowed-Params`|empty|array of non-empty strings (token) or comma-separated strings (headers)|The environment variables configured as parameters for a tokenized service via `X-Waiter-Param-*` header which will not trigger an on-the-fly mode.|Expose only environment variables which your command can handle safely in this list. You can also choose to provide default values for these parameters in your `env` configuration.|
-|`X-Waiter-Authentication`|standard|disabled or standard|The authentication mechanism to use for incoming requests.|By default, Waiter authenticates incoming requests using the standard protocol (e.g. Kerberos). If you would prefer that Waiter not authenticate incoming requests, set this flag to disabled.|
+|`X-Waiter-Authentication`|standard|disabled, standard or <authenticator name>|The authentication mechanism to use for incoming requests.|By default, Waiter authenticates incoming requests using the standard protocol (e.g. Kerberos). If you would prefer that Waiter not authenticate incoming requests, set this flag to disabled. You can specify the name of the particular authenticator you want to use, e.g. saml.|
```

Comment:
Is it possible to make all authenticators part of the composite authenticator? Then this field becomes just authenticator name.

Reply:
I think "disabled" should continue to be handled at the authentication middleware level

Reply:
done